core/config/constant.go

```diff
@@ -18,4 +18,5 @@ const (
 	DefaultMetricLogSingleFileMaxSize  uint64 = 1024 * 1024 * 50
 	DefaultMetricLogMaxFileAmount      uint32 = 8
 	DefaultSystemStatCollectIntervalMs uint32 = 1000
+	DEFAULT_COLD_FACTOR                uint32 = 3
```

Comment:
Here we may follow the spec described in https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps